.snippets/code/tutorials/main.ts

```diff
@@ -0,0 +1,108 @@
+import {In} from 'typeorm'
+import {assertNotNull} from '@subsquid/evm-processor'
+import {TypeormDatabase} from '@subsquid/typeorm-store'
+import * as erc20 from './abi/erc20'
+import {Account, Transfer} from './model'
+import {Block, CONTRACT_ADDRESS, Log, Transaction, ProcessorContext, processor} from './processor'
+
+processor.run(new TypeormDatabase({supportHotBlocks: true}), async (ctx) => {
+    ctx.log.info('Processor started');
+    let transfers: TransferEvent[] = []
+
+    ctx.log.info(`Processing ${ctx.blocks.length} blocks`);
+    for (let block of ctx.blocks) {
+        ctx.log.debug(`Processing block number ${block.header.height}`);
+        for (let log of block.logs) {
+            ctx.log.debug(`Processing log with address ${log.address}`);
+            if (log.address === CONTRACT_ADDRESS && log.topics[0] === erc20.events.Transfer.topic) {
+                ctx.log.info(`Transfer event found in block ${block.header.height}`);
+                transfers.push(getTransfer(ctx, log))
+            }
+        }
+    }
+
+    ctx.log.info(`Found ${transfers.length} transfers, processing...`);
+    await processTransfers(ctx, transfers)
+    ctx.log.info('Processor finished');
```

Comment:
these logs add a lot of noise. maybe just add a note saying you can use `ctx.log.info` to log to the terminal for debugging purposes

Reply:
Done! removed the debug stuff and added a bit on it to the debugging your squid section.